src/pages/studyView/virtualStudy/VirtualStudy.tsx

```diff
@@ -174,8 +174,9 @@ export default class VirtualStudy extends React.Component<IVirtualStudyProps, {}
 
     readonly genes = remoteData({
         invoke: async () => {
-            if(!_.isEmpty(this.entrezIds)){
-                return client.fetchGenesUsingPOST({geneIdType: "ENTREZ_GENE_ID", geneIds: this.entrezIds as any});
+            if(!_.isEmpty(this.geneIds)){
+                // not all geneIds are ENTREZ_GENE_ID
+                return client.fetchGenesUsingPOST({geneIdType: "HUGO_GENE_SYMBOL", geneIds: this.geneIds as any});
```

Comment:
I'm actually confused, I thought the gene id represents entrezGeneId or HugoSymbol. But this api call does not reflect on that.

Reply:
its a complete different API. not part of study view API's